doc/developer.texinfo

```diff
@@ -2026,7 +2028,15 @@ will take at least two hours --- but could be much more depending on the number
 different platforms on which you test the release.
 
 @noindent
-NOTE: The default version of the Java JDK on the majority of the CSE
+@c The following few lines are a texinfo hack to get IMPORTANT in hightlighted font.
```

Comment:
I don't understand the goal, and the use of fake URLs is weird to me.  I don't think we should be using `@ref`, `@anchor`, and such except for cross-references.
Why not just use `@strong` if you want emphasis?  (Same comment elsewhere.)